src/ipc/CMakeLists.txt

```diff
@@ -35,6 +35,10 @@ set_property(TARGET data_structs APPEND
 	     PROPERTY COMPILE_DEFINITIONS
 	     CC_OPTIMIZE_FLAGS="${optimization_flag}")
 
-add_local_sources(data_structs cc_version.c)
+add_local_sources(data_structs
+			cc_version.c
+			probe_support.c
+		)
```

Comment:
Please remove tabs. In other cmake files we dont indent ending parenthesis and inner arguments are just indented by 1 tab (no vertical alignment).